src/Microsoft.AspNetCore.Server.IIS/HttpProtocol.cs

```diff
@@ -302,14 +303,14 @@ public unsafe void CreateResponseHeader(bool appCompleted)
                 NativeMethods.http_set_response_status_code(_pHttpContext, (ushort)StatusCode, pReasonPhrase);
```

Comment:
Those are pretty awkward method names, even for C.

Reply:
@davidfowl

Reply:
What's that?

Reply:
Should these names be PascalCase?